README.md

```diff
@@ -406,19 +429,54 @@ To view the test results, go to `http://localhost:8888`.
 rake jasmine:ci
 ```
 
+## Jasmine for React JavaScript
+
+```
+cd client/
+yarn test:watch
+```
+
 # Static Code Analysis
 
 These tools helps us to find bugs and ensure quality without having to execute code.
 
-## JSHint
+## JSHint for Rails JavaScript
 
 ```
 bundle exec rake jshint
 ```
 
 You can read about JSHint [here](http://jshint.com/docs/).
 
-## Rubocop
+## ESLint for React JavaScript
+
+```
+cd client/
+yarn eslint app
+```
+
+You can read about ESLint [here](https://eslint.org/).
+
+## Flow for React JavaScript
+
+```
+cd client/
+yarn flow
+```
+
+You can read about Flow [here](https://flow.org/en/).
+
+## NPM Packages
+Some NPM packages have flow type enabled but fail the flow checks (e.g. radium) because it relies on an older Flow version compared to the main project" after "(e.g. radium). You'll want to put the package path under the `[ignore]` section of `.flowconfig`, for example:
```

Comment:
Oops, you copied a little too many characters 😅: " after "(e.g. radium).

Reply:
LMAO my bad. It's a long day.